packages/@aws-cdk/app-delivery/lib/pipeline-deploy-stack-action.ts

```diff
@@ -51,7 +51,7 @@ export interface PipelineDeployStackActionProps {
    *
    * @default A fresh role with admin or no permissions (depending on the value of `adminPermissions`).
    */
-  role?: iam.Role;
+  role?: iam.IRole;
```

Comment:
The change in props is not a breaking change (code that passed `Role` here would still work)